org.eclipse.wildwebdeveloper.xml/src/org/eclipse/wildwebdeveloper/xml/internal/ui/preferences/XMLPreferenceConstants.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Red Hat Inc. and others.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Angelo ZERR (Red Hat Inc.) - initial implementation
+ *******************************************************************************/
+package org.eclipse.wildwebdeveloper.xml.internal.ui.preferences;
+
+import org.eclipse.jface.util.PropertyChangeEvent;
+
+public class XMLPreferenceConstants {
+
+	public static final String XML_PREFERENCES_CATAGLOGS = "wildwebdeveloper.xml.catalogs";

```

Comment:
Why keeping the `wildwebdeveloper` prefix here? the preference story should already be bound to the bundle.

Reply:
Today catalogs is stored with this preference, but I can change that if you wish.

Reply:
fixed.